modules/dxp/apps/sharepoint-soap/sharepoint-soap-repository/src/main/java/com/liferay/sharepoint/soap/repository/connector/internal/util/test/SharepointConnectionTestUtil.java

```diff
@@ -80,9 +78,14 @@ private static String _getSharepointVMHostname() throws Exception {
 		try (BufferedReader bufferedReader = new BufferedReader(
 				new InputStreamReader(httpURLConnection.getInputStream()))) {
 
-			Stream<String> payloadStream = bufferedReader.lines();
+			StringBuilder sb = new StringBuilder();
+			String line;
 
-			String payload = payloadStream.collect(Collectors.joining());
+			while ((line = bufferedReader.readLine()) != null) {
+				sb.append(line);
+			}
+
+			String payload = sb.toString();
```

Comment:
used StringBuilder here, and remove the code 
```
if (sb.index() > 1) {
    sb.setIndex(sb.index() - 1);
}
```
because we didn't add extra `StringPool.BLANK` as before